MinVer/build/MinVer.targets

```diff
@@ -62,8 +62,10 @@
       <MinVerBuildMetadata Condition="$(MinVerVersion.Contains(`+`))">$(MinVerVersion.Split(`+`, 2)[1])</MinVerBuildMetadata>
       <AssemblyVersion>$(MinVerMajor).0.0.0</AssemblyVersion>
       <FileVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).0</FileVersion>
-      <PackageVersion>$(MinVerVersion)</PackageVersion>
-      <Version>$(MinVerVersion)</Version>
+      <InformationalVersion>$(MinVerVersion)</InformationalVersion>

```

Comment:
The one thing that comes to mind about directly setting `InformationalVersion` here is how it interacts with SourceLink, which adds the commit to the `InformationalVersion`.

Seems like we should check to make sure that still works properly with this change.

Reply:
Good point. I’ll try to add a test for that.

Reply:
@bording I've added the test and it seems to work.